src/wallet/rpcdump.cpp

```diff
@@ -115,7 +115,7 @@ UniValue importprivkey(const JSONRPCRequest& request)
     if (request.params.size() > 2)
         fRescan = request.params[2].get_bool();
 
-    if (fRescan && fPruneMode)
+    if (fRescan && pruneMode)
```

Comment:
As you already have to change every line on which the variable occurs anyhow, I'd prefer explicitly comparing the enumeration against a value instead of using it like a boolean: e.g. `pruneMode != PruneMode::NONE`. This makes it  clear to developers reading the code that this is an enumeration and not a boolean and can avoid them introducing silly bugs.